### PR TITLE
Remove PBuffer support

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: eq/server/loader.y [hunk 1]
```diff
@@ -1332,7 +1332,11 @@ IATTR:
     | EQTOKEN_VERTICAL   { $$ = eq::fabric::VERTICAL; }
     | EQTOKEN_WINDOW     { $$ = eq::fabric::WINDOW; }
     | EQTOKEN_FBO        { $$ = eq::fabric::FBO; }
-    | EQTOKEN_PBUFFER    { $$ = eq::fabric::PBUFFER; }
+    | EQTOKEN_PBUFFER
+    {
+        $$ = eq::fabric::FBO;
+        LBERROR << "Using FBO for obsolete pbuffer token" << std::endl;
```
Comment (hernando) [inline eq/server/loader.y]: "PBuffers support removed, using FBO"?

Comment (eile) [inline eq/server/loader.y]: done

**Comment by eile** [comment]
> What does it happen now if I pass -8 though the environmental variable to choose the drawable type?

Largely undefined behaviour, but from a first glance of the implementation you'll at least get some type of warning from the systemwindow.

**Comment by eile** [comment]
> All the config files that you have removed should be modified to use FBOs instead? Or you don't really want to keep them?

The removed once where PBuffer-specific. There are others which moved from PBuffer to FBO.

**Pull request merged**